src/App.tsx

```diff
@@ -33,23 +33,50 @@ let initialState: CardsDataDTO[] = [
 ];
 
 const App = () => {
-    const [data, setData] = useState(initialState)
+    const [data, setData] = useState(initialState);
+    const [isEdit, setIsEdit] = useState(false);
+    const [isOpen, setIsOpen] = useState(false);
 
-    const addCardHandler = ( title: string, text: string) => {
+    const addCardHandler = (title: string, text: string) => {
+        setData((prevData) => {
+        return [...prevData, { title: title, description: text, id: uuidv4() }];
+        });
+    };
 
-        setData((prevData)=>{
-            return [
-                ...prevData,
-                {title:title, description: text, id: uuidv4()}
-            ]
-        })
-    }
+    const removeCardHandler = (id: string) => {
+        let filteredData = data.filter((el) => el.id !== id);
+
+        setData(filteredData);
+    };
+
+    const editCardHandler = (id: string) => {
+        setIsEdit(true);
+    };
+
+    const openModal = () => {
```

Comment:
This should be one method like `toggleModal` based on `prevState`.

Reply:
done